src/runtimes/node/dynamic_imports/parser.js

```diff
@@ -0,0 +1,140 @@
+const { join, relative, resolve } = require('path')
+
+const acorn = require('acorn')
+
+const ECMA_VERSION = 2017
```

Comment:
Why not 2021?

Reply:
I was aligning this with the default esbuild target, but these are separate things in reality. There's no harm in allowing acorn to parse the latest ES version. Done in https://github.com/netlify/zip-it-and-ship-it/pull/521/commits/9717f5b2f20fea75c06fddef23c78f45ff3505c8. Thanks!